commercial/app/views/hosted/guardianAmpHostedGalleryImage.scala.html

```diff
@@ -0,0 +1,40 @@
+@import common.commercial.hosted.HostedGalleryImage
+@import views.support.ImgSrc
+@import common.commercial.hosted.HostedGalleryPage
+
+@(picture: model.ImageMedia, image: HostedGalleryImage, index: Int, page: HostedGalleryPage)
+
+@picture.largestImage.map { largestImage =>
```

Comment:
'largestImage' is defined here: https://github.com/guardian/frontend/blob/master/common/app/model/Element.scala#L63 and it does take the master image first.